TYPO3.Neos/Classes/TYPO3/Neos/Command/SiteCommandController.php

```diff
@@ -63,6 +67,88 @@ class SiteCommandController extends CommandController
     protected $systemLogger;
 
     /**
+     * @Flow\Inject
+     * @var ContextFactoryInterface
+     */
+    protected $nodeContextFactory;
+
+    /**
+     * @Flow\Inject
+     * @var NodeTypeManager
+     */
+    protected $nodeTypeManager;
+
+    /**
+     * @Flow\Inject
+     * @var NodeService
+     */
+    protected $nodeService;
+
+    /**
+     * Create a new (blank) site
+     *
+     * This command allows to create a blank site with just a single empty document. The name of the site and the
+     * packageKey to assign to it must be specified.
+     *
+     * If no "--nodeName" is specified the command will create a unique node-name from the name of the site. If a node name is given
+     * it has to be unique.
+     *
+     * The specified "--nodeType" must it exists and have the superType "TYPO3.Neos:Document".
+     *
+     * If the flag "--active" is set the created site will be activated immediately.
+     *
```

Comment:
should it maybe mention that the site will only be created in one dimension being the default dimension?